Reflex/src/classes_def.xml

```diff
@@ -1,11 +1,18 @@
 <lcgdict>
   <class name="std::vector<TLorentzVector>"/>
   <class name="std::vector<std::vector<TLorentzVector> >"/>
+  <class name="std::vector<math::XYZVector>"/>
+  <class name="std::vector<std::vector<math::XYZVector> >"/>
+  <class name="std::vector<math::XYZPoint>"/>
+  <class name="std::vector<std::vector<math::XYZPoint> >"/>
   <class name="std::vector<pat::Jet>"/>
   <class name="std::vector<std::vector<pat::Jet> >"/>
+  <class name="std::vector<std::vector<bool> >"/>
   <class name="edm::PtrVector<pat::PackedCandidate>"/>
   <class name="edm::Wrapper<std::vector<TLorentzVector> >"/>
   <class name="edm::Wrapper<std::vector<std::vector<TLorentzVector> > >"/>
+  <class name="edm::Wrapper<std::vector<std::vector<math::XYZVector> > >"/>
```

Comment:
were you going to remove these as redundant?

Reply:
The vector<vector<object>> weren't redundant, only the vector<object> ones.